notebooks/intro_to_datalad.ipynb

```diff
@@ -1171,8 +1171,8 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Read up more about the YODA principles and the `yoda` configuration in the `yoda`\n",
-    "section.\n",
+    "Read up more about the YODA principles and the `yoda` configuration in the \n",
+    "[`yoda` section of the DataLad Handbook](https://handbook.datalad.org/en/latest/basics/101-127-yoda.html).\n",
```

Comment:
we have some trickery to centralize such "shortcuts" for the cases when sections would get renumbered etc
```suggestion
    "[`yoda` section of the DataLad Handbook](https://handbook.datalad.org/en/latest/r.html?yoda).\n",
```